src/calculateRentalCost.js

```diff
@@ -4,7 +4,44 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const discountUpToSix = 20;
+  const discountSevenMore = 50;
+
+  if (days <= 0) {
+    return 'You need to rent for at least one day';
+  }
+
+  if (days <= 2) {
+    return days * pricePerDay;
```

Comment:
The condition `days <= 2` is correct, but you could simplify the logic by removing the `else` keyword because you have a `return` statement inside the `if` block.